src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { Link, useParams } from 'react-router-dom';
+import { EnrichPerson } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  people: EnrichPerson[];
+};
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { people } = props;
+  const { personSlug } = useParams();
+
+  const isPersoninList = (slug: string | undefined) => {
```

Comment:
Consider renaming `isPersoninList` to `isPersonInList` to follow the camelCase convention typically used in JavaScript and TypeScript for function names.